Sources/SwiftParser/Statements.swift

```diff
@@ -124,8 +124,10 @@ extension Parser {
       return label(self.parseContinueStatement(continueHandle: handle), with: optLabel)
     case (.fallthroughKeyword, let handle)?:
       return label(self.parseFallthroughStatement(fallthroughHandle: handle), with: optLabel)
-    case (.forgetKeyword, let handle)?:
-      return label(self.parseForgetStatement(forgetHandle: handle), with: optLabel)
+    case (.forgetKeyword, let handle)?:  // NOTE: support for deprecated _forget
+      fallthrough
+    case (.discardKeyword, let handle)?:
+      return label(self.parseDiscardStatement(discardHandle: handle), with: optLabel)
```

Comment:
The more swifty way to do write this would be 

```suggestion
     case (.forgetKeyword, let handle)?, 
         (.discardKeyword, let handle)?:
      return label(self.parseDiscardStatement(discardHandle: handle), with: optLabel)
```